Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 
 name = "nickel"
-version = "0.12.0"
+version = "0.13.0"
```

Comment:
Thank you for this. Let's make the version "0.12.1". I've not yet released 0.12 to crates.io yet, so there is no backward compatibility to break yet.

Reply:
Done. Amended as this is only a constant change